frontend/kesaseteli/handler/src/components/form/HandlerForm.tsx

```diff
@@ -74,12 +78,44 @@ const HandlerForm: React.FC<Props> = ({ application }) => {
 
   const statusId = React.useMemo(() => String(status || 'submitted'), [status]);
 
+  const additionalInfoReasons = React.useMemo(
+    () =>
+      (additional_info_user_reasons ?? []).map((reason) =>
+        t(`common:reasons.${reason}`)
+      ),
+    [additional_info_user_reasons, t]
+  );
+
+  const additionalInfoProvided = React.useMemo(
+    () => status === 'additional_information_provided',
+    [status]
+  );
+
   return (
     <>
       <Field
         id="receipt_confirmed_at"
         value={convertToUIDateAndTimeFormat(receipt_confirmed_at)}
       />
+      <$GridCell $colSpan={1} $rowSpan={additionalInfoProvided ? 11 : 7}>
+        <$Notification
+          data-testid="vtj-info"
+          label={t(`common:vtj.title`)}
+          type="info"
+        >
+          <pre>
+            {JSON.stringify(
+              {
+                name: 'Anneli kulmala',
+                hetu: '010105-T123',
+                address: '12345 Vantaa',
+              },
+              null,
+              2
+            )}
```

Comment:
Is this data here on purpose..?

Reply:
Yes, there is mock data for the demo purpose until we get the real data out.